pkg/helm/version_detect_test.go

```diff
@@ -14,6 +14,8 @@ func Test_DetectHelmVersion(t *testing.T) {
 
 		_ = os.Setenv("HELM3", "yes")
 		defer os.Unsetenv("HELM3")
+		_ = os.Setenv("HELM_POST_RENDERER_PATH", "testdata/helm-fake/helm3/helm")
```

Comment:
Maybe [this](https://pkg.go.dev/testing@master#T.Setenv) fits better here

Reply:
It could be. But it would be rather matter of refactoring of  all tests.